src/src/WaterPokemon.java

```diff
@@ -0,0 +1,32 @@
+public class WaterPokemon extends Pokemon {
+    int defence;
+
+
+    public WaterPokemon(String name, int weight, int hp, String food, String sound, int defence) {
+        super(name, weight, hp, food, sound);
+        this.defence = defence;
+    }
+
+    @Override
+    public void eats(String food) {
+        System.out.println(name + " eats " + food);
+    }
```

Comment:
al je Override methodes zijn het zelfde. dan had je net zo goed deze SOUT in je abstracte 'Pokemon' class kunnen plaatsen. Het heeft pas zin om overwrite te maken als je return van de methode verschilt.
 Bijv bij waterpokemon:  (name + "drinks" + food);
bij Fire: (name + "burns" + food)